toss/Trajectory.py

```diff
@@ -79,7 +84,7 @@ def integrate(self, x: np.ndarray) -> Union[np.ndarray, float, float]:
             dt = self.time_step, 
             rtol = 1e-12, 
             atol = 1e-12,
-            constants=dict(risk_zone_radius = self.radius_bounding_sphere)) #, mesh_vertices = self.mesh_vertices, mesh_faces = self.mesh_faces
+            constants=dict(risk_zone_radius = self.radius_bounding_sphere))
```

Comment:
I am not sure this syntax works as you intend. This passes a dictionary containing the bounding sphere under the key with value of risk_zone_radius, I think?

E.g. if `risk_zone_radius = 2.5`, and `radius_bounding_sphere = 5.0` then it will be {2.5 : 5.0}.